src/fsd/4-entities/character/data/generateTraitsJson.sh

```
@@ -0,0 +1,29 @@
+# The input files are:
+# - ./newCharacterData.json
+# - ./newNpcData.json
+# The output file is:
+# - ./traits-generated.json
+
+# The output file will contain a JSON object where each key is a trait key found.
+# The corresponding value will be null since we don't have the user-facing name or icon yet.
+# The reasons for this instead of is because TypeScript treats keys as string literals but
+# values as more variable types (e.g. string | null), so this format is more useful for type definitions.
+
+#!/bin/bash
```

Shellcheck (0.11.0)

[error] The shebang must be on the first line. Delete blanks and move comments.

(SC1128)